.github/workflows/ci.yml

```diff
@@ -122,3 +122,33 @@ jobs:
       working-directory: ${{github.workspace}}/build
       shell: bash -l {0}
       run: ctest --output-on-failure -C $BUILD_TYPE
+
+  # Plan for preoducing code coverage report
```

Comment:
```suggestion
  # Job producing code coverage report
```

Reply:
that's what I get fro copy-pasting... :P